components/boards/AllArticleList.tsx

```diff
@@ -4,24 +4,32 @@ import AllArticleItem from "./AllArticleItem";
 import { Article, ArticleApiData } from "@/types/articleTypes";
 import SearchInput from "@/components/layout/SearchInput";
 import Button from "@/components/layout/Button";
-import Dropdown from "@/components/layout/Dropdown";
 import Link from "next/link";
 import { getArticle } from "@/lib/articleApi";
 import { useRouter } from "next/router";
+import SortDropdown from "../layout/Dropdown/SortDropdown";
+import {
+  defaultOrderType,
+  ORDER_TYPE_ENUM,
+  orderTypeKeysKR,
+  orderTypeKR,
+  orderTypeUS,
+} from "@/constants/orderConstants";
 
 export default function AllArticleList({
   initialArticles,
 }: {
   initialArticles: Article[];
 }) {
   const [articles, setArticles] = useState(initialArticles);
-  const [orderBy, setOrderBy] = useState<ArticleApiData["orderBy"]>("recent");
+  const [orderBy, setOrderBy] = useState(ORDER_TYPE_ENUM.RECENT);
```

Comment:
아래와 같이 수정하면 defaultOrderType에 대한 서비스에서 기본적인 정렬 방식을 어떻게 할지에 대한 정책이 바뀌어도 수정이 편할 것 같아요.

```suggestion
  const [orderBy, setOrderBy] = useState(defaultOrderType);
```

Reply:
앗 그러네요. defaultOrderType를 만들어놓고 안쓰다니...!! defaultOrderType로 쓰면 말씀하신대로 기본 정렬 정책이 바껴도 하나하나 수정할 필요 없으니 좋네요.😆